cli/dcoscli/config/main.py

```diff
@@ -109,6 +109,38 @@ def _unset(name):
 
     return 0
 
+def _format_config(file_value, effective_value, name=None, envvar_name=None,
+                   left_pad_fmt=None):
```

Comment:
Currently the password case is in the _show function, because it's special logic as regards value and effective_value, which means duplication, or just going back to the inline '%-35s', not sure which is least ugly.  I'll duplicate the vars too if that seems best.